src/crossChain/AxelarProvider.ts

```diff
@@ -69,11 +83,53 @@ export class AxelarProvider implements CrossChainProvider {
 
     // @ts-ignore
     // tslint:disable-next-line
-    const txResponse = (await this.squid.executeRoute({
-      route: route.route,
-      signer
-    })) as ethers.providers.TransactionResponse
+    let txResponse: ethers.providers.TransactionResponse | null = (await this.squid.executeRoute(
+      {
+        route: route.route,
+        signer
+      }
+    )) as ethers.providers.TransactionResponse
 
+    const originChainTxHash = txResponse.hash
+
+    // if it's an actual cross-chain interaction, we need to get the tx hash in the destination chain
+    if (route.route.params.fromChain !== route.route.params.toChain) {
+      let status: StatusResponse | undefined
+      try {
+        status = await this.squid.getStatus({
+          transactionId: txResponse.hash,
+          integratorId: INTEGRATOR_ID,
+          requestId: route.requestId
+        })
+      } catch (error) {
+        console.error('error: ', error)
+      }
+      txResponse = null
+      const destinationChain = (route.route.params
+        .toChain as unknown) as ChainId
```

Comment:
Is `toChain` a string? `ChainId` is of type number, we might need a `toString()` here:
```suggestion
        .toChain as unknown) as ChainId
```